app/main.py

```diff
@@ -1,3 +1,73 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int | float | Distance) -> Distance:
+        if isinstance(other, (int, float)):
+            return Distance(
+                km=self.km + other
+            )
+
+        return Distance(
+            km=self.km + other.km
+        )
+
+    def __iadd__(self, other: int | float | Distance) -> Distance:
+        if isinstance(other, (int, float)):
+            self.km += other
+            return self
+
+        self.km += other.km
+        return self
+
+    def __mul__(self, other: int | float) -> Distance:
+        if isinstance(other, (int, float)):
+            return Distance(
+                km=self.km * other
+            )
+
+    def __truediv__(self, other: int | float) -> Distance:
+        if isinstance(other, (int, float)):
+            return Distance(
+                km=round(self.km / other, 2)
+            )
+
+    def __lt__(self, other: int | float | Distance) -> bool:
+        if isinstance(other, (int, float)):
+            return self.km < other
+
+        return self.km < other.km
+
+    def __gt__(self, other: int | float | Distance) -> bool:
+        if isinstance(other, (int, float)):
+            return self.km > other
+
+        return self.km > other.km
+
+    def __eq__(self, other: int | float | Distance) -> bool:
+        if isinstance(other, (int, float)):
```

Comment:
you could simplify these methods with not + previously implemented operators. Or more advanced if you use [functools.total_ordering](https://docs.python.org/3/library/functools.html#functools.total_ordering)